frontend/src/pages/JobFeed/JobFeed.tsx

```diff
@@ -6,8 +6,7 @@ import FilterDrawer from "./components/FilterDrawer";
 import JobCard from "./components/JobCard";
 import JobPreviewCard from "./components/JobPreviewCard";
 import { ArrowDropUp, ArrowDropDown } from "@mui/icons-material";
-// @ts-ignore
-import { JobPost } from "../../../../../backend/node_modules/prisma/prisma-client";
+import {JobPost} from "../../interfaces/JobPost"
 
 interface JobFeedResponseType {
   data: JobPost[];
```

Comment:
Do you mean that both `import { JobPost } from "../../../../../backend/node_modules/prisma/prisma-client";` and
`import {JobPost} from "../../interfaces/JobPost"` are problematic because they have a Date (non-string) field for a json value? If that is the case, then the frontend should have its own interfaces instead of getting the interface from the backend.

Reply:
They're the same, yes. But they're not necessarily problematic. I think they actually are good types to have around! However, they *don't* represent the type of the response from the API requests.